internal/server/statvar/golden/search_statvar/crime.json

```diff
@@ -21759,24 +21777,6 @@
           "dcid": "Victim_Count_CriminalIncidents_BiasMotivationRace_CrimeAgainstSociety_IsHateCrime_AsianAlone"
         }
       ]
-    },
-    {
-      "dcid": "dc/g/HateCrimeIncidents_BiasMotivation-race_CrimeType-CrimeAgainstSociety_TargetedRace-BlackOrAfricanAmericanAlone",
```

Comment:
There is no update to FBI HateCrime yet, so don't expect these deletions.

Reply:
This seems to be replaced by addition of other star vars. @chejennifer is this possible due to search results not stable?

Reply:
it looks like the stat vars that replaced this are ranked "higher" and looks like they were recently added so should make sense? https://critique.corp.google.com/cl/599147687

Reply:
Got it. Thanks for checking!